### PR TITLE
feat(ado-improvements-1): Remove unnecessary auth input from ADO extension

### PR DESCRIPTION
**Pull request by ThanyaLeif**
#### Details

This PR removes unnecessary auth input after comments in PRs were disabled from ADO Extension #1041 , since the only function of this parameter was to provide a service connection to contribute to the target repo.

Image of the inputs before this PR, including "Azure Repo Connection" field
image

Image of the inputs after this PR, "Azure Repo Connection" field was removed
image

##### Motivation

Feature work.

##### Context
The service connection is only used to grant comment writing permissions to the extension. Behavior that was remove in PR #1041 

#### Pull request checklist

- [ n/a ] Addresses an existing issue: Fixes #0000
- [x] Added relevant unit test for your changes. (`yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`yarn precheckin`)
- [n/a] (after PR created) The `Accessibility Checks (pull_request)` check should fail. All other checks should pass.

### CODE REVIEW

**Comment by dbjorge** [comment]
Do we know whether this is a breaking change or not? ie, have you verified what happens if you have an existing build that uses this parameter, and then the extension updates to not require it anymore? If it breaks, how do we indicate a breaking change to users? If it doesn't break, should we warn users that are still passing the connection that it isn't necessary anymore?

**Comment by ThanyaLeif** [comment]
> Do we know whether this is a breaking change or not? ie, have you verified what happens if you have an existing build that uses this parameter, and then the extension updates to not require it anymore? If it breaks, how do we indicate a breaking change to users? If it doesn't break, should we warn users that are still passing the connection that it isn't necessary anymore?

Yes, I verified that the old builds that use this parameter do not break after introducing the new changes.
However, users are not yet aware of this new behavior, I will discuss this with the team.

**Comment by karanbirsingh** [comment]
> > Do we know whether this is a breaking change or not? ie, have you verified what happens if you have an existing build that uses this parameter, and then the extension updates to not require it anymore? If it breaks, how do we indicate a breaking change to users? If it doesn't break, should we warn users that are still passing the connection that it isn't necessary anymore?
> 
> @dbjorge Yes, I verified that the old builds that use this parameter do not break after introducing the new changes. However, users are not yet aware of this new behavior, I will discuss this with the team.

We discussed this a little further today. It seems old client usage will still work (ADO ignored the input in our test), but we agreed it's semantically a big enough change (no more comments) that bumping the major version makes sense. That way users will know to remove any service connections or manual auth they've added.

We've added an ADO workitem 1893587 that covers deprecation. This might include publishing an update to 1.x with upgrade instructions before bumping the overall version to 2.x. We propose doing that in a separate PR.

File: packages/ado-extension/src/task-config/ado-task-config.spec.ts [hunk 1]
```diff
@@ -30,30 +30,29 @@ describe(ADOTaskConfig, () => {
...
-        ${'scanTimeout'}               | ${'100000'}         | ${100000}                                               | ${() => taskConfig.getScanTimeout()}
-        ${'localhostPort'}             | ${'8080'}           | ${8080}                                                 | ${() => taskConfig.getLocalhostPort()}
-        ${'localhostPort'}             | ${undefined}        | ${undefined}                                            | ${() => taskConfig.getLocalhostPort()}
-        ${'repoServiceConnectionName'} | ${'testName'}       | ${'testName'}                                           | ${() => taskConfig.getRepoServiceConnectionName()}
-        ${'baselineFile'}              | ${'./baselineFile'} | ${getPlatformAgnosticPath(__dirname + '/baselineFile')} | ${() => taskConfig.getBaselineFile()}
-        ${'failOnAccessibilityError'}  | ${true}             | ${true}                                                 | ${() => taskConfig.getFailOnAccessibilityError()}
-        ${'singleWorker'}              | ${true}             | ${true}                                                 | ${() => taskConfig.getSingleWorker()}
```
Comment (karanbirsingh) [inline packages/ado-extension/src/task-config/ado-task-config.spec.ts:56]: I was confused why the whole table was changed (as opposed to just the line). I realized that we are removing the longest field name. This means the whitespace for each line decreased by 1 space so the | pipe symbols are aligned.

**Review by karanbirsingh — APPROVED** [review]

File: packages/ado-extension/src/task-config/ado-task-config.spec.ts (hunk 1 above)
Comment (ThanyaLeif) [reply packages/ado-extension/src/task-config/ado-task-config.spec.ts:56]: Yeah, this looks weird at the beginning, but as you mentioned, the formater automatically changes the arrangement of the whole table when removing the longest field name.

**Pull request merged**